pkg/render/intrusion_detection.go

```diff
@@ -545,3 +553,53 @@ func (c *intrusionDetectionComponent) globalAlertTemplates() []runtime.Object {
 		},
 	}
 }
+
+func (c *intrusionDetectionComponent) intrusionDetectionPodSecurityPolicy() *policyv1beta1.PodSecurityPolicy {
+	psp := basePodSecurityPolicy()
+	psp.GetObjectMeta().SetName("intrusion-detection")
+	return psp
+}
+
+func (c *intrusionDetectionComponent) intrusionDetectionPSPClusterRole() *rbacv1.ClusterRole {
+	return &rbacv1.ClusterRole{
+		TypeMeta: metav1.TypeMeta{Kind: "ClusterRole", APIVersion: "rbac.authorization.k8s.io/v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "intrusion-detection-psp",
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				// Allow access to the pod security policy in case this is enforced on the cluster
+				APIGroups:     []string{"policy"},
+				Resources:     []string{"podsecuritypolicies"},
+				Verbs:         []string{"use"},
+				ResourceNames: []string{"intrusion-detection"},
+			},
+		},
+	}
+}
+
+func (c *intrusionDetectionComponent) intrusionDetectionPSPClusterRoleBinding() *rbacv1.ClusterRoleBinding {
+	return &rbacv1.ClusterRoleBinding{
+		TypeMeta: metav1.TypeMeta{Kind: "ClusterRoleBinding", APIVersion: "rbac.authorization.k8s.io/v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "intrusion-detection-psp",
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "intrusion-detection-psp",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "intrusion-detection-controller",
+				Namespace: IntrusionDetectionNamespace,
+			},
+			{
+				Kind:      "ServiceAccount",
+				Name:      "default",
```

Comment:
Why do we need default here?

Reply:
The installer job doesn't have a a service account associated with it so we need to make this accessible to the job by applying it to the `default` service account.